src/OpenLoco/Windows/Vehicle.cpp

```diff
@@ -942,6 +944,78 @@ namespace OpenLoco::Ui::Vehicle
             return self;
         }
 
+        static void cloneVehicle(window* self)
+        {
+            static loco_global<uint16_t, 0x0113642A> _113642A;
+            auto head = Common::getVehicle(self);
+            Vehicles::Vehicle train(head);
+            Vehicles::VehicleHead* newHead = nullptr;
+
+            // Get total cost for a new vehicle (note this should be removed when turned into a game command)
+            currency32_t totalCost = 0;
+            for (auto& car : train.cars)
+            {
+                auto cost = GameCommands::queryDo_5(car.front->object_id, -1);
+                if (cost == GameCommands::FAILURE)
```

Comment:
Our Linux CI is warning about signedness here. Should we just make it explicitly unsigned?

```
../src/OpenLoco/Windows/Vehicle.cpp:959:26: warning: comparison of integer expressions of different signedness: ‘int’ and ‘const uint32_t’ {aka ‘const unsigned int’} [-Wsign-compare]
  959 |                 if (cost == GameCommands::FAILURE)
      |                     ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
../src/OpenLoco/Windows/Vehicle.cpp:969:27: warning: comparison of integer expressions of different signedness: ‘OpenLoco::currency32_t’ {aka ‘int’} and ‘const uint32_t’ {aka ‘const unsigned int’} [-Wsign-compare]
  969 |             if (totalCost == GameCommands::FAILURE)
      |                 ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
../src/OpenLoco/Windows/Vehicle.cpp:978:27: warning: comparison of integer expressions of different signedness: ‘int32_t’ {aka ‘int’} and ‘const uint32_t’ {aka ‘const unsigned int’} [-Wsign-compare]
  978 |             if (regs2.ebp == GameCommands::FAILURE)
      |                 ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
```